server/services/teamHealthService.ts

```diff
@@ -297,13 +304,23 @@ export default class TeamHealthService {
       .filter(env => env.type === 'prod')
       .sort((e1, e2) => e1.buildDate.getTime() - e2.buildDate.getTime())[0]
 
+    const commitDate = component.latest_commit?.date_time && startOfDay(new Date(component.latest_commit?.date_time))
+    const commitSha = component.latest_commit?.sha?.slice(0, 7)
+
+    const baseSha = commitSha
+    const baseDate = component.part_of_monorepo ? latestDevEnvironment?.buildDate || commitDate : commitDate
```

Comment:
if mono repo use the dev env like previously.. otherwise changing one component inside a mono repo will affect all components scores.